File: src/main/java/com/naminhyeok/fantazzk/FantazzkApplication.java
package com.naminhyeok.fantazzk;

import java.util.TimeZone;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.modulith.Modulithic;

@SpringBootApplication
@Modulithic(systemName = "Fantazzk")
public class FantazzkApplication {
    public static void main(String[] args) {
        TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
        SpringApplication.run(FantazzkApplication.class, args);
    }
}

File: src/main/kotlin/com/naminhyeok/fantazzk/FantazzkApplication.kt
package com.naminhyeok.fantazzk

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.modulith.Modulithic
import java.util.TimeZone

@SpringBootApplication
@Modulithic(systemName = "Fantazzk")
class FantazzkApplication

fun main(args: Array<String>) {
    TimeZone.setDefault(TimeZone.getTimeZone("UTC"))
    runApplication<FantazzkApplication>(*args)
}
